pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools<60.0", "wheel"]
```

Comment:
Can you add a comment explaining why the setuptools version is pinned?

Reply:
Here is the issue that I referenced: https://github.com/readthedocs/readthedocs.org/issues/8775

For some reason my readthedocs build was failing remotely and pinning the version solved it. I am not sure if the issue has been resolved yet, but we can look more into it if needed.